clients/mocks_test.go

```diff
@@ -43,6 +43,16 @@ func TestGatekeeperMock_UsersInGroup(t *testing.T) {
 	}
 }
 
+func TestGatekeeperMock_GroupsForUser(t *testing.T) {
+	expected := makeExpectedUsersPermissions()
+
+	gkc := NewGatekeeperMock(nil, nil)
```

Comment:
like discussed, I think it is awkward to unit test a method through a test that is skipping the entire logic of the method. It could be better to add a new test to test the mock with expectations. Now or later :)

Reply:
tests updated